luda-editor/luda-editor-client/src/app/editor/timeline/timeline_body/track_body/camera_track_body/camera_clip_body/mod.rs

```diff
@@ -75,14 +85,48 @@ impl CameraClipBody {
         .attach_event(move |builder| {
             let clip_id = props.clip.id.clone();
             builder.on_mouse_down(move |event| {
+                let clicked_part = if is_sashes_showing {
+                    [SashDirection::Left, SashDirection::Right]
```

Comment:
```suggestion
                    [SashDirection::Right] // NOTE : No left sash yet. it's intended to be added later if it's really needed.
```
If you want disable left sash, you have to change this line like [Line.115](https://github.com/NamseEnt/namseent/pull/110/files#diff-2cfb64fdaf79f2652455bf8040caf39afdd3ed9a0fc2495db4af6623cd3303f3R115).

Reply:
I moved that variable into const variables because it's more fancyyyy
https://github.com/NamseEnt/namseent/pull/110/files#diff-2cfb64fdaf79f2652455bf8040caf39afdd3ed9a0fc2495db4af6623cd3303f3R24

Reply:
fancyyyy